war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -231,32 +231,56 @@ var FormChecker = {
   },
 
   sendRequest: function (url, params) {
-    if (params.method != "get") {
+    if (params.method !== "get") {
       var idx = url.indexOf("?");
       params.parameters = url.substring(idx + 1);
       url = url.substring(0, idx);
     }
-    new Ajax.Request(url, params);
+
+    fetch(url, {
+      method: "post",
```

Comment:
In the Prototype version it was possible to call this function with `params.method` set to `get`, and this would be plumbed through to `Ajax.Request`. Now, the same request would result in a POST request. I think we should retain compatibility by plumbing this through to the Fetch API. If for some reason it is desirable to change the behavior, then the preceding `if` statement can be simplified.

Reply:
Removed as it was unused since 2.285
https://github.com/search?q=org%3Ajenkinsci+checkMethod%3D%22get%22+language%3AXML&type=code&l=XML